suite/tests/CMakeLists.txt

```diff
@@ -2948,6 +2948,7 @@ else (UNIX)
     append_link_flags(security-win32.except-execution "/dynamicbase:no")
 
     tobuild(security-win32.except-guard security-win32/except-guard.c)
+    tobuild(security-win32.except-int2d security-win32/except-int2d.c)
```

Comment:
Yes, CL == ChangeList == nomenclature in some review systems for patch sent for review.

AppVeyor should test it for us on 64-bit.